frontend/javascripts/oxalis/model_initialization.js

```diff
@@ -313,20 +326,34 @@ function initializeSettings(initialUserSettings: Object, initialDatasetSettings:
   Store.dispatch(initializeSettingsAction(initialUserSettings, initialDatasetSettings));
 }
 
-function initializeDataLayerInstances(): {
+function initializeDataLayerInstances(
+  gpuFactor: ?number,
+): {
   dataLayers: DataLayerCollection,
   connectionInfo: ConnectionInfo,
   isMappingSupported: boolean,
   maximumDataTextureCountForLayer: number,
+  smallestCommonBucketCapacity: number,
 } {
   const { dataset } = Store.getState();
   const layers = dataset.dataSource.dataLayers;
 
-  const { textureInformationPerLayer, isMappingSupported } = validateSpecsForLayers(layers);
+  const requiredBucketCapacity =
+    constants.GPU_FACTOR_MULTIPLIER *
+    (gpuFactor != null ? gpuFactor : constants.DEFAULT_GPU_MEMORY_FACTOR);
+
+  console.log("demanding requiredBucketCapacity:", requiredBucketCapacity);
```

Comment:
Not sure whether you want to keep those two console.logs

Reply:
I removed one and kept the other so that we potentially have some hardware information available in airbrake.